READMEs/parameters.md

```diff
@@ -59,8 +58,8 @@ First, in your working directory, create `config.ini` file and fill as follows:
     network = https://rinkeby.infura.io/v3/<your infura project id>
 
     [resources]
-    aquarius.url = https://provider.rinkeby.oceanprotocol.com
-    provider.url = https://aquarius.rinkeby.oceanprotocol.com
+    metadata_cache.uri = https://aquarius.rinkeby.oceanprotocol.com
```

Comment:
should this be an underscore? or dots? maybe not a combination of the two... `metadata_cache_uri`?

Reply:
if we want to make it like provider_url... should we use "L" instead of "I"?

Reply:
I chose `metadata_cache.uri` because there are previous examples of options with underscores, yet all of the options that refer to URL/URIs separate the last part with a dot. I'm definitely open to change if there's a strong preference for something different.

My top choice for alternatives would be `metadata_cache_uri` like @calina-c suggested because it does away with the dot notation which seemed odd to me.

I used URI because the issue #229 description seemed to prefer `uri` over `url`. I think breaking the consistency between this new option and the legacy options is OK. In light of this, I think we should stick to uri for this new option. Please let me know if there's a strong opposing opinion.

Reply:
I'm open to both `metadata_cache_uri` and `metadata_cache.uri` but the second one looks like a typo, even if it isn't :D It just pops. @trentmc deferring to you for guidance here.

Reply:
@calina-c @trentmc and I agreed that `metadata_cache_uri` with all underscores was preferable.